python/cuml/test/test_dbscan.py

```diff
@@ -110,3 +143,5 @@ def test_dbscan_sklearn_comparison(name):
     assert(sk_n_clusters == cu_n_clusters)
 
     clusters_equal(sk_y_pred, cu_y_pred, sk_n_clusters)
+    if stream is not None:
```

Comment:
I think a sync is needed in any cases. See my comment above.

Reply:
Done.